app/views/ChooseProvider.js

```diff
@@ -149,13 +153,76 @@ class ChooseProviderScreen extends Component {
     }
   }
 
+  createInvalidTextPopUp() {
```

Comment:
Can these alerts be moved outside of the class into regular functions? they don't appear to use `this`.